x-pack/plugins/observability_solution/inventory/server/routes/entities/get_latest_entities.ts

```diff
@@ -28,19 +30,26 @@ export async function getLatestEntities({
   inventoryEsClient: ObservabilityElasticsearchClient;
   sortDirection: 'asc' | 'desc';
   sortField: EntityColumnIds;
-  entityTypes?: EntityType[];
+  entityTypes?: string[];
   kuery?: string;
 }) {
   // alertsCount doesn't exist in entities index. Ignore it and sort by entity.lastSeenTimestamp by default.
   const entitiesSortField = sortField === 'alertsCount' ? ENTITY_LAST_SEEN : sortField;
 
+  const from = `FROM ${ENTITIES_LATEST_ALIAS}`;
+  const where: string[] = [getBuiltinEntityDefinitionIdESQLWhereClause()];
+
+  if (entityTypes) {
+    where.push(getEntityTypesWhereClause(entityTypes));
```

Comment:
Now the `entityTypes` could be any string that the user passes, maybe it's time to use the params. Up to you

Reply:
I tried, but it was throwing an error. I was having another look at the types now and apparently I missed something... will try it again. Thanks.

Reply:
https://github.com/elastic/kibana/pull/197292/commits/a03061e1132fa268e60470297c505298ffec8eaa